src/EventSubscriber/KernelRequestSubscriber.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * Contao Facebook Import Bundle for Contao Open Source CMS
+ *
+ * @copyright  Copyright (c), Moritz Vondano
+ * @license    MIT
+ * @link       https://github.com/m-vo/contao-facebook-import
+ *
+ * @author     Moritz Vondano
+ */
+
+namespace Mvo\ContaoFacebookImport\EventSubscriber;
+
+use Contao\CoreBundle\Routing\ScopeMatcher;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\HttpKernel\Event\RequestEvent;
+use Symfony\Component\HttpKernel\KernelEvents;
+
+class KernelRequestSubscriber implements EventSubscriberInterface
```

Comment:
I think I would favor a listener over a subscriber. This way you can easily reconfigure it in your app. Wdyt?

Also let's name it, so that you already know what it is doing by looking at the file. What about  `BackendAssetListener` or something?

Reply:
Well, it was just copy & paste from the docs - see: https://docs.contao.org/dev/guides/adding-back-end-assets/#adding-assets-globally. I'm not sure what would be the best way or kind of philosophy 😁

Reply:
You cannot easily reconfigure an event subscriber, that's why I think listeners should be favored in bundles. The docs make sense in an app context IMHO.

see https://symfony.com/doc/current/event_dispatcher.html#listeners-or-subscribers

Reply:
Done 😁